src/lib/footnotes-utils.ts

```diff
@@ -0,0 +1,140 @@
+/*!
+ * © 2024 Atypon Systems LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { FootnoteWithIndex } from '@manuscripts/style-guide'
+import { skipTracking } from '@manuscripts/track-changes-plugin'
+import { isFootnoteNode, ManuscriptNode, schema } from '@manuscripts/transform'
+import { Fragment } from 'prosemirror-model'
+import { findChildrenByType, NodeWithPos } from 'prosemirror-utils'
+import { EditorView } from 'prosemirror-view'
+
+export const getNewFootnotePos = (
+  footnotesElement: NodeWithPos,
+  tablesFootnoteLabels: Map<string, string | undefined> | undefined,
```

Comment:
As far as I can see this is passed but not used.

Reply:
removed